yogstation.dme

```diff
@@ -694,6 +694,7 @@
 #include "code\game\gamemodes\hivemind\hivemind.dm"
 #include "code\game\gamemodes\hivemind\objectives.dm"
 #include "code\game\gamemodes\hivemind\radar.dm"
+#include "code\game\gamemodes\kings\Kings.dm"
```

Comment:
you might want to put it on lowercase also try moving to the yogstation\code folder, as being modularised will make it more likely to get in + yogs only gamemode

Reply:
okay